ts/collection.ts

```diff
@@ -377,4 +387,16 @@ export class Collection<T extends any> {
     );
     return parse(docs) as Array<T & WithID>;
   }
+
+  // Limit
+  public limit(maxLimit: number) {
+    this.maxQueryLimit = maxLimit;
+    return this;
+  }
+
+  // Skip
+  public skip(skipCount: number) {
+    this.skipDocCount = skipCount;
+    return this;
+  }
```

Comment:
```ts
const users = db.collection<UserSchema>("users");
const docs1 = users.skip(2).limit(10).find();

// The execution of this line of code will be incorrectly affected.
const docs2 = users.find();
```

We can only put limit and skip into the second parameter of the find method. Or the find method returns a special promise after execution, which has limit and skip methods.

Reply:
If I got you correctly, you're expecting something like this right?
```js
const users = db.collection<UserSchema>("users");
const docs1 = users.find({}, { skip: 2, limit: 10 });
// Or
const docs2 = users.find().skip(2).limit(10);
```

Reply:
Yes correct